Extension/src/LanguageServer/configurations.ts

```diff
@@ -1893,7 +1893,7 @@ export class CppProperties {
         }
 
         // Resolve and split any environment variables
-        paths = this.resolveAndSplit(paths, undefined, this.ExtendedEnvironment);
+        paths = this.resolveAndSplit(paths, undefined, this.ExtendedEnvironment, true);
```

Comment:
@bobbrow I don't understand -- this is the assumeRelative parameter -- it appears you're "fixing" the squiggle by disabling squiggling for all paths.

Reply:
@browntarik Actually, I don't see any change/fix from this change? I see it exit out of resolveAndSplit before it even examines the 4th bool arg.

Reply:
You're right, it looks like the real fix is to resolve and split glob patterns before validating config paths